app/controllers/container_topology_controller.rb

```diff
@@ -5,7 +5,13 @@ class ContainerTopologyController < ApplicationController
   after_action :set_session_data
 
   def show
-
+    # When navigated here without id, it means this is a general view for all providers (not for a specific provider)
+    # all previous navigation should not be displayed in breadcrumbs as the user could arrive from
+    # any other page in the application.
+    if params[:id].nil?
+      @breadcrumbs.clear
+    end
```

Comment:
> prefer more terse @breadcrumbs.clear if params[:id].nil?

I find it less readable this way.
I read this from the end of the sentence starting from the condition :)
But if that's the only thing standing between this and a merge... :)